paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/EmptyFormSpec.kt

```diff
@@ -0,0 +1,3 @@
+package com.stripe.android.paymentsheet.elements
+
+internal object EmptyFormSpec : FormItemSpec()
```

Comment:
Can this just be a form with an empty list?

Reply:
understood this is needed for completeFormFieldValues to return

Reply:
Can you add some more details into the use of this, and if it should ever be used by those creating LayoutSpec, or if it is really intended to just be internal forever.  I believe it is the later.